tidb-cloud/changefeed-sink-to-mysql.md

```diff
@@ -26,8 +26,8 @@ This document describes how to stream data from TiDB Cloud to MySQL using the **
 Before creating a changefeed, you need to complete the following prerequisites:
 
 - Set up your network connection
-- Export and load the full data
-- Create a target table in MySQL
+- Export and load the existing data (optional)
+- Create corresponding target tables in MySQL
```

Comment:
Create corresponding target tables in MySQL （If you don‘t load the existing data and only replicate on going changes）